internal/pkg/manifest/env.go

```diff
@@ -107,6 +108,32 @@ type environmentVPCConfig struct {
 	Subnets subnetsConfiguration `yaml:"subnets,omitempty"`
 }
 
+type securityGroupConfig struct {
+	Ingress []securityGroupRule `yaml:"ingress,omitempty"`
+	Egress  []securityGroupRule `yaml:"egress,omitempty"`
+}
+
+// IsEmpty returns true if there is no security group rule defined.
+func (cfg securityGroupConfig) IsEmpty() bool {
```

Comment:
Does this method need to be exported?

Reply:
Actually it was required before when we had not implemented the Demeter law suggested by Efe but now I think we don't need to export this I will change it to lower case.